src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: ostapDanyliv
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: id
```

Comment:
We don't need a separate `id` column for this table. Make user_id + role_id a composite PK to disallow duplicate entries and remove `id` column. (You can edit this script until it pushed to main).